src/styles/all-styles/card.scss

```diff
@@ -0,0 +1,107 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  width: (4 * $card-width) + (3 * $card-margin-right) + (2 * $padding-horizont);
+  padding: $padding-vertical $padding-horizont;
+  margin: 0 auto;
```

Comment:
As was mentioned try to avoid external geometry styling on a block level. For this case you can create some kind of container class and corresponding element class container__cards. There you can define external geometry styles
if something is not clear to you feel free to ask in fe_chat channel, mentors will help you to figure out this approach